autoroutes.pyx

```diff
@@ -239,10 +288,44 @@ cdef class Node:
                 self.pattern = pattern
                 self.regex = re.compile(pattern)
 
+    cdef Node insert(self, str path):
+        cdef:
+            Node node
+            Edge edge = None
+            str prefix
+            int bound, end
+            unsigned int nb_slugs
+
+        node = self.common_edge(path)
+
+        if node:
+            return node
+
+        nb_slugs = path.count('{')
+        start = path.find('{')
```

Comment:
I wonder if `path.split('{', 1)` is faster here.

Reply:
humm, but we need an index, not a path.
Plus this is during the tree construction, so there is no need for absolute speed (I mean not at any cost :) ).